.github/workflows/build-documentation.yml

```diff
@@ -0,0 +1,22 @@
+name: build-documentation
```

Comment:
Consider gathering it with clang-format into one workflow.

Reply:
merged